components/GameCard.tsx

```diff
@@ -21,79 +20,69 @@ export const GameCard: React.FC<GameCardProps> = ({
 }) => {
   return (
     <Box
-      border="3px solid black"
-      borderBottom="5px solid black"
-      borderRight="5px solid black"
-      p={8}
+      p={0}
       transition="background 0.3s ease"
-      w="360px"
+      w="480px"
+      borderBottom="5px solid rgba(255,255,255,0.2)"
+      pb={'20px'}
     >
+      <Link
+        alignItems="center"
+        display="inline-flex"
+        fontSize="sm"
+        gap={2}
+        href={`${EXPLORER_URLS[chainId]}/address/${id}`}
+        isExternal
+      >
+        {shortenAddress(id)}
+      </Link>
       <NextLink href={`/games/[gameId]`} as={`/games/${id}`}>
         <Box
-          h="116px"
+          mb={10}
           transition="background 0.3s ease"
           _hover={{
             h2: {
-              borderBottom: '2px solid black',
+              color: 'accent',
             },
           }}
         >
           <Heading
-            borderBottom="2px solid transparent"
             display="inline-block"
-            fontSize="3xl"
-            mb={4}
-            transition="all 0.3s ease"
-            variant="primary"
+            fontSize="40px"
+            fontWeight="normal"
+            lineHeight="40px"
           >
             {name}
           </Heading>
-          <Text fontSize="sm">{shortenText(description, 60)}</Text>
+          {/* <Text fontSize="sm">{shortenText(description, 60)}</Text> */}
```

Comment:
I'm personally against removing the description for games. I imagine people will want to understand at a high-level what the game is about without having to click into each one

Reply:
k, cool, will give it a good place on the card!